components/Chat.tsx

```diff
@@ -0,0 +1,55 @@
+import { View, Text, StyleSheet, Image } from 'react-native';
+import React from 'react';
+
+type ChatProps = {
+  Name: string;
+  Image: string;
+  LastMessage: string;
+};
+
+export default function Chat(props: ChatProps) {
+  const pic = {
+    uri: props.Image,
+  };
```

Comment:
Why not just do inline destructuring?

```suggestion
export default function Chat({ Image }: ChatProps) {
  
```